### ISSUE TITLE
Frozen setter always freezes the player

### ISSUE THREAD

**Issue by xxshady** [altmp/altv-issues#1462]
### Description of the problem

/

### Reproduction steps

```js
player.frozen = false // not always but this can freeze player

// and this will not unfreeze it
player.frozen = true
player.frozen = false
```

### Expected behaviour

/

### Operating system

Windows 11

### Version

dev/10.0-dev21

**Comment by deluvas1911** [comment]
Confirmed on dev/10.0-dev21

**Comment by YannBcf** [comment]
I want to add that this issue seems extremely random, I could also reproduce it but now it doesn't happen at all for me, so there must be a specific factor

**Comment by YannBcf** [comment]
Fixed in dev/11.0-dev15
